### PR TITLE
ROX-18793: Add changelog entry with monitoring

### PR DESCRIPTION
**Pull request by janisz**
## Description

Add changelog entry with monitoring

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

CI

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Added Features
 
 - Telemetry collection enabled by default for self-managed installations. Opt-out is available on bundle generation, or at any time via the System Configuration UI.
+- OpenShift Container Platform monitoring is configured and enabled by default for OpenShift 4 installations. Disabling is available on bundle generation, or at any time via the System Configuration UI.
```
Comment (stehessel) [inline CHANGELOG.md:13]: > System Configuration UI

There is a toggle in the operator CR UI, but not in the Central UI. I think we just should mention the Helm/CR flags here that may be used to disable it.

**Review by stehessel — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Added Features
 
 - Telemetry collection enabled by default for self-managed installations. Opt-out is available on bundle generation, or at any time via the System Configuration UI.
+- Integration with OpenShift Container Platform monitoring is configured and enabled by default for OpenShift 4 installations. The flag `monitoring.openshift.enabled: false` disables the integration.
```
Comment (0x656b694d) [inline CHANGELOG.md:13]:
```suggestion
- Integration with OpenShift Container Platform monitoring is configured and enabled by default for OpenShift 4 installations. The option `monitoring.openshift.enabled: false` disables the integration.
```

**Review by 0x656b694d — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 2 above)
Comment (janisz) [reply CHANGELOG.md:13]: @stehessel WDYT?

Comment (stehessel) [reply CHANGELOG.md:13]: Sounds good to me

**Pull request merged**